.gitignore

```diff
@@ -7,3 +7,4 @@ Makefile
 node_modules/
 examples/node/svgdump/
 examples/node/pdf2png/*.png
+.idea
```

Comment:
Revert this change (and therefore this commit). We don't keep editor-specific lines in the `.gitignore` file; it is the responsibility of the developer to setup these directories elsewhere or exclude them when committing. Moreover, we try to keep the changes in a patch limited to only the intended functionality to make review easier.